### PR TITLE
nav2 header for folder/filepreview screens

### PR DESCRIPTION
**Pull request by songgao**
image

image

image

image

### CODE REVIEW

File: shared/fs/common/open-chat.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+import * as Util from '../../util/kbfs'
+import * as React from 'react'
+
+type OwnProps = {|
+  path: Types.Path,
+|}
+
+const OpenChat = props =>
+  props.onChat && (
+    <Kb.WithTooltip text="Chat with users in this folder">
```
[the remark this answers is not in the document]
Comment (songgao) [reply shared/fs/common/open-chat.js]: This was actually copied over from old code. It does seem fine to me, though I'm OK with changing it to "Chat with users in this team" if that sounds better to you?

cc @jzila who composed this in the first place.

**Comment by songgao** [comment]
> If I try to view an image on mobile w/ nav2 this happens:

Yeah this PR only deals with desktop. I'll fix mobile in a separate PR.

File: shared/fs/common/open-chat.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (jzila) [reply shared/fs/common/open-chat.js]: Yeah I guess team would be better.

**Comment by songgao** [comment]
Turns out mobile needs to use a different header component which doesn't seem to be there yet, so I just went ahead and make it usable on nav2 for now. Will implement the actual mobile design when that header component is ready.

@buoyad ready for another look!

**Comment by songgao** [comment]
Did some additional tweaks for mobile. I'm not sure how we are gonna approach the dynamically shrinkable header, and the small version doesn't work for Files screens, so I just did the big version for now.

Some screenshots:

image

image

**Pull request merged**